rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -2705,9 +2705,8 @@ public Coin getEstimatedFeesForNextPegOutEvent() throws IOException {
 
         int pegoutRequestsCount = getQueuedPegoutsCount();
 
-        if (activations.isActive(ConsensusRule.RSKIP385) && pegoutRequestsCount == 0){
-            pegoutRequestsCount = 1;
-        } else if (!activations.isActive(RSKIP271) || pegoutRequestsCount == 0) {
+        if (!activations.isActive(ConsensusRule.RSKIP385) &&
+                (!activations.isActive(RSKIP271) || pegoutRequestsCount == 0)) {
             return Coin.ZERO;
```

Comment:
To me, this looks harder to follow and understand the logic behind it. When I see the return at the end `return Coin.ZERO` I assume that this act as an "else", meaning this is the default logic, which I don't think represents the real business logic. I prefer to keep the `Coin.ZERO` isolated as a specific old condition that we keep just because of the history of the blockchain.